File: SDKs/googleapis.com/content/v2.1/python/src/sdk/models/shared/accountstatusstatistics.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json


@dataclass_json
@dataclass
class AccountStatusStatistics:
    active: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'active' }})
    disapproved: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'disapproved' }})
    expiring: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'expiring' }})
    pending: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'pending' }})
    

File: SDKs/googleapis.com/content/v2.1/typescript/src/sdk/models/shared/accountstatusstatistics.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";


export class AccountStatusStatistics extends SpeakeasyBase {
  @Metadata({ data: "json, name=active" })
  active?: string;

  @Metadata({ data: "json, name=disapproved" })
  disapproved?: string;

  @Metadata({ data: "json, name=expiring" })
  expiring?: string;

  @Metadata({ data: "json, name=pending" })
  pending?: string;
}
